### PR TITLE
Raccoon screen

### PR DESCRIPTION
**Pull request by Donnyd88**
Added the raccoon screen. It's a separate .py file so, changing it later to add the assets as they become available shouldn't affect other files and we can merge now and keep the changes more frequent and smaller.

### CODE REVIEW

**Comment by Donnyd88** [comment]
@Techwise-Team-15/everyone updated current branch should be ready for merge. Review when able.

File: assets/items.png
Comment (dinaolmelak) [inline assets/items.png:1]: We would need this `PNG` @Donnyd88 😅 I am sorry we would need this back

File: main_menu.py
Comment (dinaolmelak) [inline main_menu.py:1]: Awesome 👍

File: raccoon_screen.py
Comment (dinaolmelak) [inline raccoon_screen.py:1]: Glad to have `raccoon_screen`

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Be sure to add the `assets/items.png` file that we had. Btw look under `Files changed` under your PR and you will see what @RuthAnn1  has added for the new items. @Donnyd88

**Review by dinaolmelak — COMMENTED** [review]
Awesome 👍🏽

**Review by dinaolmelak — APPROVED** [review]
LGTM

**Pull request merged**